src/HttpClient/Response.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Sentry\HttpClient;
+
+class Response
```

Comment:
What's the point of removing `final` here? This class is a value object and it doesn't need to be extended by third parties